### PR TITLE
Changed the facilityValue to value to keep BSS and GEF consistent

### PR DESCRIPTION
**Pull request by IgnatG**
- Changed the `facilityValue` property to `facility` to keep BSS and GEF consistent

### CODE REVIEW

File: dtfs-central-api/src/v1/swagger-definitions/portal/facility-ewcs.js [hunk 1]
```diff
@@ -120,7 +120,7 @@
  *       premiumType:
  *         type: string
  *         example: 'In advance'
- *       oremiumFrequency:
+ *       premiumFrequency:
```
Comment (ttbarnes) [inline dtfs-central-api/src/v1/swagger-definitions/portal/facility-ewcs.js:123]: whoa, good spot

**Review by ttbarnes — COMMENTED** [review]
*Great* work, bonus points for fixing the typos(!) Can approve when the tests are passing 👍

**Review by abhi-markan — CHANGES_REQUESTED** [review]
Can you please resolve the conflicts, but great work!

**Review by ttbarnes — APPROVED** [review]
LGTM, great work!

**Review by abhi-markan — APPROVED** [review]

**Pull request merged**